roles/atomic_upgrade/tasks/main.yml

```diff
@@ -34,10 +34,9 @@
 
 - block:
 
-    - name: Workaround BZ1290281
-      selinux:
-        policy: targeted
-        state: permissive
+    - name: Workaround old atomic command bug: failure b/c /tmp is a symlink
```

Comment:
Not really... as a monkey who's simply on the user side of things, I don't know or care about /tmp being a symlink. All I know is that when I run atomic upgrade, I see a barf message with something about fsetxattr, and the solution is `setenforce 0`.  At least, I think that's why you're doing this, right? Anyway, the point is, if a future maintainer decides to delete this code, and the fsetxattr barf reappears, having "fsetxattr" in the description would be a really helpful way to realize that it's not yet time to delete it.